python_modules/dagster/dagster/_core/storage/upath_io_manager.py

```diff
@@ -52,26 +51,53 @@ def get_metadata(
         """Child classes should override this method to add custom metadata to the outputs."""
         return {}
 
+    # Read/write operations on paths can generally be handled by methods on the
+    # UPath class, but when the backend requires credentials, this isn't
+    # always possible. Override these path_* methods to provide custom
+    # implementations for targeting backends that require authentication.
+
+    def path_rm(self, path: UPath) -> None:
```

Comment:
Maybe just call this `unlink` or `unlink_path`? In general, I don't think we should use abbreviations like `rm` unless directly referencing something in a lower-level library.

Reply:
ok, `unlink`